portfolio-stylesheet.css

```diff
@@ -0,0 +1,663 @@
+	Media Queries
+   =============================================
+*/
+
+@media screen and (max-width: 1350px) {
+	.pageTitle {
+		font-size: 45px;
+	}
+}
+
+@media screen and (max-width: 1270px) {
+	.gallery-container {
+		grid-gap: 5px;
+		grid-template-columns: 1fr 1fr;
+	}
+}
+
+@media screen and (max-width: 1100px) {
+	.middle-aboutme .about-grid {
+		grid-template-columns: 1fr 1fr 1fr;
+	}
+}
+
+@media screen and (max-width: 1080px) {
+	nav {
+		display: block;
+	}
+}
+
+@media screen and (max-width: 1170px) {
+	.pageTitle {
+		font-size: 25px;
+	}
+	nav a {
+		font-size: 10px;
+		padding: 0 5px 0 5px;
+	}
+}
+
+@media screen and (max-width: 1060px) {
+	.top-aboutme {
+		height: 300px;
+	}
+	.top-aboutme img {
+		border-radius: 50%;
+		width: 200px;
+		height: 200px;
+	}
+	.aboutme-heading h1 {
+		font-size: 40px;
+	}
+	.top-aboutme p {
+		font-size: 25px;
+	}
+	.middle-aboutme {
+		padding-top: 50px;
+	}
+	.contact-image img {
+		width: 400px;
+		height: auto;
+	}
+}
+
+/* This is for the portfolio gallery which has the hover function.
+Because there is no hover on a touch device, I need to move the text
+to the bottom of the image and display it by default. */
+
+/*@media only screen and (max-width: 1024px) {
+	.text-wrapper {
+		opacity: 1;
+		justify-content: flex-end;
+		text-shadow: 1px 1px 1px #000;
+	}
+	.project-title {
+		margin-bottom: 1em;
+	}
+}*/
+
+@media only screen and (max-width: 960px) {
+	.contact-image img {
+		width: 250px;
+		height: auto;
+	}
+	.top-contact-container .contact-intro-text {
+		padding-left: 20px;
+	}
+	.top-contact-container .contact-image {
+		padding-right: 20px;
+	}
+}
+
+@media screen and (max-width: 900px) {
+	header {
+		background-image: none;
+		background-color: #dbdbdd;
+	}
+}
+
+@media screen and (max-width: 860px) {
+	.gallery-container {
+		grid-template-columns: 1fr;
+		justify-content: center;
+		text-align: center;
+	}
+}
+
+@media screen and (max-width: 800px) {
+	nav a {
+		display: block;
+	}
+	.cv-details {
+		width: 600px;
+	}
+	.top-contact-container .contact-image {
+		padding-left: 20px;
+	}
+}
+
+@media screen and (max-width: 780px) {
+	.top-aboutme p {
+		display: flex;
+		flex-wrap: wrap;
+		font-size: 20px;
+		padding-right: 15px;
+	}
+}
+
+@media screen and (max-width: 670px) {
+	.home h2 {
+		font-size: 40px;
+	}
+	.header {
+		padding: 250px 0;
+	}
+	.middle-aboutme .about-grid {
+		grid-template-columns: 1fr 1fr;
+	}
+}
+
+@media screen and (max-width: 610px) {
+	.cv-details {
+		width: 400px;
+	}
+}
+
+@media screen and (max-width: 500px) {
+  .navBar a {
+    display: block;
+  }
+  .pageTitle {
+	  display: block;
+  }
+  .home h2 {
+	  display: block;
+  }
+  .about-me {
+	  display: block;
+  }
+  .websites {
+	  display: block;
+  }
+  .contact-me {
+	  display: block;
+  }
+  .before-footer {
+	  display: flex;
+	  align-items: center;
+  }
+  .footer {
+	  display: block;
+  }
+}
+
+@media screen and (max-width: 300px) {
+  .navBar a {
```

Comment:
if you need to apply the same property and value to multiple elements you can do this:
```
.element1,
.element2,
.element3 {
    display: block;
}
```